### DISCUSSION TITLE
NFT metadata not visible

### DISCUSSION THREAD

**Discussion by yusuphoo** [Cyfrin/foundry-full-course-cu#1443] [Q&A] [answered]
Please how can i view my image metadata after importing it to ipfs i only see the image?

**Comment by EngrPips** [comment, marked as answer]
Hey @yusuphoo  i assume we always first upload the actual image to ipfs then get the address of the image on ipfs so as to use it in the image metadata that we would be creating for that image, after creating the image metadata then we upload it to ipfs as well. So doing it like that we would get back the image metadata address on ipfs and whenever we open the image metadata address on ipfs the image address is always a part of the metadata information.

I would say you should ensure that you are visiting the address of the image metadata on ipfs and not that of the image itself.

**Reply by yusuphoo** [reply]
Thanks for making it more clear to me.

**Reply by EngrPips** [reply]
I am glad i could be of help
